accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerHeader.scala

```diff
@@ -16,14 +16,15 @@
 
 package geotrellis.spark.io.accumulo
 
-import geotrellis.spark.io.LayerHeader
+import geotrellis.spark.io.{LayerHeader, LayerType, AvroLayerType}
 
 import spray.json._
 
 case class AccumuloLayerHeader(
   keyClass: String,
   valueClass: String,
-  tileTable: String
+  tileTable: String,
+  layerType: LayerType = AvroLayerType
```

Comment:
@pomadchin Ah, that's a good catch!

Reply:
@pomadchin Instead of making it `None`, what if we just check to see if the `LayerHeader` has the `layerType` field when reading, and if doesn't just assume that it's an `AvroLayerType`? Since any layer that doesn't have the `layerType` field would be Avro based (it wouldn't be possible to read old COG layers after this PR is merged).

```scala
    def read(value: JsValue): LayerHeader =
      value.asJsObject.getFields("format", "keyClass", "valueClass", "layerType") match {
        case Seq(JsString(_format), JsString(_keyClass), JsString(_valueClass), JsString(_layerType)) =>
          new LayerHeader {
            val format = _format
            val keyClass = _keyClass
            val valueClass = _valueClass
            def layerType = LayerType.fromString(_layerType)
          }
        case Seq(JsString(_format), JsString(_keyClass), JsString(_valueClass)) =>
          new LayerHeader {
            val format = _format
            val keyClass = _keyClass
            val valueClass = _valueClass
            def layerType = AvroLayerType
          }
        case _ =>
throw new DeserializationException(s"LayerHeader expected, got: $value")
```